### PR TITLE
Refactor assemble function and add support to H rune just as example

### PR DESCRIPTION
**Pull request by thiagonache**

### CODE REVIEW

File: gmachine.go [hunk 1]
```diff
@@ -53,6 +53,10 @@ var PredefinedConstants = map[string]Word{
 	"STDERR":  PortStderr,
 }
 
+var Runes = map[string]Word{
+	"H": 72,
```
Comment (bitfield) [inline gmachine.go:57]: I don't think we want to have to define all Unicode runes in a map, do we? I just checked and there are 120,803 of them!

Suppose we interpret any single character in single quotes as a rune value, and just use `rune(s)` to convert it. We could also treat any sequence of characters inside double quotes as a list of runes.

File: gmachine.go [hunk 2]
```diff
@@ -172,52 +176,109 @@ func (g *GMachine) ExecuteBinary(binPath string) error {
...
 			}
-			word = Word(value)
-
+			words = append(words, word)
+			continue
 		}
-		words = append(words, word)
+		//fmt.Println(token, "is opcode")
+		words = append(words, instruction.Opcode)
 		if instruction.Operands > 0 {
```
Comment (bitfield) [inline gmachine.go:202]: It's feeling like this is now getting so deeply nested it's worth moving out to a function, isn't it?

**Review by bitfield — COMMENTED** [review]
🇧🇷

File: gmachine.go (hunk 1 above)
Comment (thiagonache) [reply gmachine.go:57]: > I don't think we want to have to define all Unicode runes in a map, do we? I just checked and there are 120,803 of them!
hahahah that would be funny... newbies.... hheh


> 
> Suppose we interpret any single character in single quotes as a rune value, and just use `rune(s)` to convert it. We could also treat any sequence of characters inside double quotes as a list of runes.
this sounds like a good idea.

File: gmachine.go (hunk 2 above)
Comment (thiagonache) [reply gmachine.go:202]: sounds reasonable to me.

Comment (thiagonache) [reply gmachine.go:216]: started by `AssembleData` function

**Pull request merged**